README.md

```diff
@@ -1188,40 +1188,40 @@ Use this parameter — especially when using the `fuzzy_match` parameter — to
 
 * **instance_name**
```

Comment:
I changed the wording of this section quite a bit to how I understood, please check it's still accurate.